src/OpenTelemetry.Instrumentation.AspNet.TelemetryHttpModule/CHANGELOG.md

```diff
@@ -9,6 +9,12 @@
   by [custom
   implementations](https://github.com/open-telemetry/opentelemetry-dotnet/tree/main/docs/trace/extending-the-sdk#sampler).
   ([#1871](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1871))
+* `TelemetryHttpModule` will now pass the `http.request.method` tag
```

Comment:
please merge this into the previous entry, so we don't need to duplicate most of the contents

Reply:
fixed in [4c3c59a](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/2023/commits/4c3c59a957154b687e29ccb72c76ec55fcfd504a)